FHEM/59_Buienradar.pm

```diff
@@ -476,20 +477,20 @@ sub Define {
     my ( $hash, $def ) = @_;
     $global_hash = $hash;
 
-    my @a = split( '[ \t][ \t]*', $def );
-    my $name = $a[0];
+    my @arguments = split( '[ \t][ \t]*', $def );
+    my $name = $arguments[0];
     my $latitude;
     my $longitude;
     my $language = lc ::AttrVal('global', 'language', 'DE');
 
-    if ( ( int(@a) == 2 ) && ( ::AttrVal( 'global', 'latitude', -255 ) != -255 ) )
+    if ( ( int(@arguments) == 2 ) && ( ::AttrVal( 'global', 'latitude', -255 ) != -255 ) )
```

Comment:
Was möchtest Du mit int(@arguments) machen?
Soll das die Anzahl an Elementen in dem Array liefern?
Wenn ja, dann wäre ich gerade nicht sicher ob das so geht ;)

Ich kenne da scalar (@arguments) oder halt $#arguments :)

Reply:
```
perl -e '@a = (a, b, c); print int(@a);'
3
```
Hast aber recht, `scalar` oder `$#` finde ich auch besser lesbar.

Reply:
#122